packs/outlaws-of-alkenstar-bestiary/clearwater-cleaner.json

```diff
@@ -404,7 +404,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p><strong>Requirements</strong> The Clearwater Cleaner is adjacent to a crate</p>\n<hr />\n<p><strong>Effect</strong> The Clearwater Cleaner picks up a crate and heaves it up to 15 feet. Upon landing, the crate breaks open in a @Template[burst|distance:5]. Each creature within the area takes @Damage[2d6[bludgeoning]] damage (@Check[reflex|dc:13|basic]), and the area becomes difficult terrain until cleared.</p>"
+                    "value": "<p><strong>Requirements</strong> The Clearwater Cleaner is adjacent to a crate</p>\n<hr />\n<p><strong>Effect</strong> The Clearwater Cleaner picks up a crate and heaves it up to 15 feet. Upon landing, the crate breaks open in a @Template[burst|distance:5]. Each creature within the area takes @Damage[(2d6)[bludgeoning]|options:area-damage] damage (@Check[reflex|dc:13|basic|options:area-effect]), and the area becomes difficult terrain until cleared.</p>"
```

Comment:
```suggestion
                    "value": "<p><strong>Requirements</strong> The Clearwater Cleaner is adjacent to a crate</p>\n<hr />\n<p><strong>Effect</strong> The Clearwater Cleaner picks up a crate and heaves it up to 15 feet. Upon landing, the crate breaks open in a @Template[burst|distance:5]. Each creature within the area takes @Damage[2d6[bludgeoning]|options:area-damage] damage (@Check[reflex|dc:13|basic|options:area-effect]), and the area becomes difficult terrain until cleared.</p>"
```

Reply:
The parentheses aren't needed unless it's a more complicated expression, so they're typically omitted. See https://github.com/foundryvtt/pf2e/wiki/Style-Guide#inline-check-links